invoicenet/common/util.py

```diff
@@ -27,6 +27,18 @@
 import pytesseract
 from pytesseract import Output
 
+import os
+from PIL import Image
+from googleapiclient.discovery import build
+import base64
+from google.cloud import vision
+
+## API keys for google ocr
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"]="file.json"
```

Comment:
Could you rename `file.json` to something more relevant? Something like `google_api_keys.json`.